File: funnyvote/app/src/main/java/com/heaton/funnyvote/ui/about/aboutapp/AboutAppContract.java
package com.heaton.funnyvote.ui.about.aboutapp;

import com.heaton.funnyvote.BasePresenter;
import com.heaton.funnyvote.BaseView;

public interface AboutAppContract {
    interface View extends BaseView<Presenter> {
        void showShareApp();
    }
    interface Presenter extends BasePresenter{
        void shareApp();
    }
}

File: funnyvote/app/src/main/java/com/heaton/funnyvote/ui/about/aboutapp/AboutAppContract.kt
package com.heaton.funnyvote.ui.about.aboutapp

import com.heaton.funnyvote.BasePresenter
import com.heaton.funnyvote.BaseView

interface AboutAppContract {
    interface View : BaseView<Presenter> {
        fun showShareApp()
    }

    interface Presenter : BasePresenter {
        fun shareApp()
    }
}
